Source/WebKit/UIProcess/ios/ProcessAssertionIOS.mm

```diff
@@ -44,7 +44,7 @@
 
 static WorkQueue& assertionsWorkQueue()
 {
-    static NeverDestroyed<Ref<WorkQueue>> workQueue(WorkQueue::create("ProcessAssertion Queue"));
+    static NeverDestroyed<Ref<WorkQueue>> workQueue(WorkQueue::create("ProcessAssertion Queue", WorkQueue::QOS::UserInitiated));
```

Comment:
Seems OK, but wouldn't UserInteractive (which is higher priority than UserInitiated) be better? An assertion can be taken on behalf of a user interaction.

Reply:
We are using a sync API, with UserInteractive, we should guarantee to not block for a long time otherwise the OS might reduce the priority itself.
acquireSync might take some time so it is best to stick with UserInitiated.